cypress/e2e/sign_in.js

```diff
@@ -0,0 +1,91 @@
+import {sigh_in_selectors as sigh_in_selectors} from "../selectors/sigh_in_selectors";
```

Comment:
spaces should be added inside of the name exported item like:
import { sigh_in_selectors } as sigh_in_selectors from "../selectors/sigh_in_selectors";

Reply:
Added spaces inside of the exported name, please review